Dockerfile-classic-api

```diff
@@ -0,0 +1,44 @@
+# arxiv/classic-api
+#
+# Defines the runtime for the arXiv classic API, which provides a metadata
+# query API backed by Elasticsearch.
+
+FROM arxiv/base:0.16.6
+
+WORKDIR /opt/arxiv
+
+# Install MySQL.
+# RUN yum install -y which mysql mysql-devel
+
+# Add Python application and configuration.
+ADD classic-api.py /opt/arxiv/
+ADD Pipfile /opt/arxiv/
+ADD Pipfile.lock /opt/arxiv/
+RUN pip install -U pip pipenv
+RUN pipenv install
+
+ENV PATH "/opt/arxiv:${PATH}"
+
+ADD schema /opt/arxiv/schema
+ADD mappings /opt/arxiv/mappings
+ADD search /opt/arxiv/search
+ADD wsgi-classic-api.py config/uwsgi-classic-api.ini /opt/arxiv/
+
+ENV LC_ALL en_US.utf8
```

Comment:
Since each ENV makes a layer, move these to the start of the Dockerfile.

Reply:
please see updated version